tests/core/workflow/test_run.py

```
@@ -0,0 +1,362 @@
+
+
+class TestWorkflowRunEvents:
+    """Tests for workflow event emission."""
+
+    @pytest.mark.asyncio
+    async def test_events_emitted(self):
+        """Events are emitted during workflow execution."""
+        session = Session(name="test")
+        events = []
+
+        async def capture_event(event):
```

Ruff (0.14.10)

Missing return type annotation for private function `capture_event`

Add return type annotation: `None`

(ANN202)